addons/web_editor/static/src/js/wysiwyg/wysiwyg.js

```diff
@@ -395,8 +406,14 @@ const Wysiwyg = Widget.extend({
                     this._peerToPeerLoading.then(() => this.ptp.handleNotification(payload));
                 }
             }
-        });
+        }
+        this.call('bus_service', 'addEventListener', 'notification', collaborationBusListener);
         this.call('bus_service', 'addChannel', this._collaborationChannelName);
+        this._collaborationStopBus = () => {
+            Wysiwyg.activeCollaborationChannelNames.delete(this._collaborationChannelName);
+            this.call('bus_service', 'removeEventListener', 'notification', collaborationBusListener);
+            this.call('bus_service', 'deleteChannel', this._collaborationChannelName);
+        }
```

Comment:
This also looks like a bugfix that was squashed with the rest. I suppose the listeners weren't properly removed before this PR either ?